bin/release

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Update the Formula for a new version.
+#
+# Usage: bin/release VERSION
+#
+# Requires sha1sum be installed.
+#
+###
+set -e
+
+set -- 0.2.11
```

Comment:
I thought I had removed that. It sets arguments so I could test the script without having to explicitly pass them.